### ISSUE TITLE
[TwitCastingLive] Exception when requesting JSON of /show endpoint: ERROR: object of type 'NoneType' has no len()

### ISSUE THREAD

**Issue by nurupo** [yt-dlp/yt-dlp#2835]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.02.04**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Description

Recently I have started getting:

```
ERROR: object of type 'NoneType' has no len()
```

errors when running yt-dlp on `/show` pages, e.g.

```
yt-dlp -j https://twitcasting.tv/vazz_kun/show/
```

The above still prints JSON, but also outputs the error.

Note that the command might take very long to run, depending on how many pages the /show section has. So when testing, it might be a good idea to find a streamer with as fewer pages in /show as possible. /show must also contain videos that have "REC" on them and have no lock icon, as otherwise yt-dlp wouldn't find anything and wouldn't output anything.

### Verbose log

```shell
[debug] Command-line config: ['-vU', '-j', 'https://twitcasting.tv/vazz_kun/show/']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8 (No ANSI), err utf-8 (No ANSI), pref UTF-8
[debug] yt-dlp version 2022.02.04 [c1653e9ef]
[debug] Python version 3.9.2 (CPython 64bit) - Linux-5.10.0-11-amd64-x86_64-with-glibc2.31
[debug] exe versions: ffmpeg 4.3.3-0, ffprobe 4.3.3-0
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
Latest version: 2022.02.04, Current version: 2022.02.04
yt-dlp is up to date (2022.02.04)
[debug] [TwitCastingUser] Extracting URL: https://twitcasting.tv/vazz_kun/show/
[download] Downloading playlist: vazz_kun - Live History
[TwitCastingUser] vazz_kun: Downloading page 1
[TwitCastingUser] playlist vazz_kun - Live History: Downloading 2 videos
[download] Downloading video 1 of 2
[debug] [TwitCasting] Extracting URL: https://twitcasting.tv/vazz_kun/movie/721662456
[TwitCasting] 721662456: Downloading webpage
[TwitCasting] 721662456: Downloading live info
[download] Downloading playlist: Live #721662456
[TwitCasting] playlist Live #721662456: Collected 1 videos; downloading 1 of them
[download] Downloading video 1 of 1
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 721662456-0: Downloading 1 format(s): 0
{"id": "721662456-0", "url": "https://dl193234.twitcasting.tv/tc.vod.v2/v1/streams/721662456.0.2/hls/master.m3u8?k=%2Ftc.vod%2Fv%2F721662456.0.2-1645323146-1645351946-844d9e6d-7b7e7ddd8723ec8d", "ext": "mp4", "protocol": "m3u8", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.81 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Encoding": "gzip, deflate", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", "Origin": "https://twitcasting.tv", "Referer": "https://twitcasting.tv/", "Cookie": "hl=en; did=b622fb8d4fa58716f42c6a6e0debb955"}, "title": "Live #721662456", "description": "Live #721662456//", "thumbnail": "https://imagegw03.twitcasting.tv/image3/image.twitcasting.tv/mtmbs/image44_1/db2/f8/2b03b1f8.17f1496a26a.jpg", "timestamp": 1645348859, "uploader_id": "vazz_kun", "duration": 1360.377, "view_count": 11, "is_live": false, "n_entries": 1, "_last_playlist_index": 1, "playlist_count": 1, "playlist_index": 1, "playlist_autonumber": 1, "playlist": "Live #721662456", "playlist_id": "721662456", "playlist_title": "Live #721662456", "playlist_uploader": null, "playlist_uploader_id": "vazz_kun", "extractor": "TwitCasting", "webpage_url": "https://twitcasting.tv/vazz_kun/movie/721662456", "webpage_url_basename": "721662456", "webpage_url_domain": "twitcasting.tv", "extractor_key": "TwitCasting", "fulltitle": "Live #721662456", "thumbnails": [{"url": "https://imagegw03.twitcasting.tv/image3/image.twitcasting.tv/mtmbs/image44_1/db2/f8/2b03b1f8.17f1496a26a.jpg", "id": "0"}], "display_id": "721662456-0", "duration_string": "22:40", "upload_date": "20220220", "requested_subtitles": null, "__has_drm": false, "format_id": "0", "format": "0 - unknown", "resolution": null, "dynamic_range": "SDR", "epoch": 1645323150, "_filename": "Live #721662456 [721662456-0].mp4", "filename": "Live #721662456 [721662456-0].mp4", "urls": "https://dl193234.twitcasting.tv/tc.vod.v2/v1/streams/721662456.0.2/hls/master.m3u8?k=%2Ftc.vod%2Fv%2F721662456.0.2-1645323146-1645351946-844d9e6d-7b7e7ddd8723ec8d", "_type": "video"}
ERROR: object of type 'NoneType' has no len()
Traceback (most recent call last):
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1381, in wrapper
    return func(self, *args, **kwargs)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1465, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1585, in process_ie_result
    return self.__process_playlist(ie_result, download)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1800, in __process_playlist
    ie_result = self.run_all_pps('playlist', ie_result)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 3323, in run_all_pps
    info = self.run_pp(pp, info)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 3296, in run_pp
    files_to_delete, infodict = pp.run(infodict)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/postprocessor/common.py", line 22, in run
    ret = func(self, info, *args, **kwargs)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/postprocessor/common.py", line 117, in wrapper
    return func(self, info)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/postprocessor/ffmpeg.py", line 1160, in run
    if len(in_files) < len(entries):
TypeError: object of type 'NoneType' has no len()

[download] Downloading video 2 of 2
[debug] [TwitCasting] Extracting URL: https://twitcasting.tv/vazz_kun/movie/721661232
[TwitCasting] 721661232: Downloading webpage
[TwitCasting] 721661232: Downloading live info
[download] Downloading playlist: Live #721661232
[TwitCasting] playlist Live #721661232: Collected 1 videos; downloading 1 of them
[download] Downloading video 1 of 1
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 721661232-0: Downloading 1 format(s): 0
{"id": "721661232-0", "url": "https://dl193234.twitcasting.tv/tc.vod.v2/v1/streams/721661232.0.2/hls/master.m3u8?k=%2Ftc.vod%2Fv%2F721661232.0.2-1645323149-1645351949-844d9e6d-6eff6dd7f2c0d208", "ext": "mp4", "protocol": "m3u8", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.81 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Encoding": "gzip, deflate", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", "Origin": "https://twitcasting.tv", "Referer": "https://twitcasting.tv/", "Cookie": "hl=en; did=b622fb8d4fa58716f42c6a6e0debb955"}, "title": "Live #721661232", "description": "Live #721661232//", "thumbnail": "https://imagegw03.twitcasting.tv/image3/image.twitcasting.tv/mtmbs/image242_3/db2/30/2b03ad30.17f14809d7f.jpg", "timestamp": 1645347040, "uploader_id": "vazz_kun", "duration": 1801.117667, "view_count": 9, "is_live": false, "n_entries": 1, "_last_playlist_index": 1, "playlist_count": 1, "playlist_index": 1, "playlist_autonumber": 1, "playlist": "Live #721661232", "playlist_id": "721661232", "playlist_title": "Live #721661232", "playlist_uploader": null, "playlist_uploader_id": "vazz_kun", "extractor": "TwitCasting", "webpage_url": "https://twitcasting.tv/vazz_kun/movie/721661232", "webpage_url_basename": "721661232", "webpage_url_domain": "twitcasting.tv", "extractor_key": "TwitCasting", "fulltitle": "Live #721661232", "thumbnails": [{"url": "https://imagegw03.twitcasting.tv/image3/image.twitcasting.tv/mtmbs/image242_3/db2/30/2b03ad30.17f14809d7f.jpg", "id": "0"}], "display_id": "721661232-0", "duration_string": "30:01", "upload_date": "20220220", "requested_subtitles": null, "__has_drm": false, "format_id": "0", "format": "0 - unknown", "resolution": null, "dynamic_range": "SDR", "epoch": 1645323152, "_filename": "Live #721661232 [721661232-0].mp4", "filename": "Live #721661232 [721661232-0].mp4", "urls": "https://dl193234.twitcasting.tv/tc.vod.v2/v1/streams/721661232.0.2/hls/master.m3u8?k=%2Ftc.vod%2Fv%2F721661232.0.2-1645323149-1645351949-844d9e6d-6eff6dd7f2c0d208", "_type": "video"}
ERROR: object of type 'NoneType' has no len()
Traceback (most recent call last):
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1381, in wrapper
    return func(self, *args, **kwargs)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1465, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1585, in process_ie_result
    return self.__process_playlist(ie_result, download)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1800, in __process_playlist
    ie_result = self.run_all_pps('playlist', ie_result)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 3323, in run_all_pps
    info = self.run_pp(pp, info)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 3296, in run_pp
    files_to_delete, infodict = pp.run(infodict)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/postprocessor/common.py", line 22, in run
    ret = func(self, info, *args, **kwargs)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/postprocessor/common.py", line 117, in wrapper
    return func(self, info)
  File ".pyenv/lib/python3.9/site-packages/yt_dlp/postprocessor/ffmpeg.py", line 1160, in run
    if len(in_files) < len(entries):
TypeError: object of type 'NoneType' has no len()

[download] Finished downloading playlist: vazz_kun - Live History
```

**Comment by Lesmiscore** [comment]
This is probably fixed in ed66a17ef0b18159. Let me check

**Comment by Lesmiscore** [comment]
Works without error on post-ed66a17e revision. Closing it
```
$ ./yt-dlp.sh -vj https://twitcasting.tv/vazz_kun/show/
[debug] Command-line config: ['-vj', 'https://twitcasting.tv/vazz_kun/show/']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2022.02.04 [c1653e9ef] (source)
[debug] Lazy loading extractors is disabled
[debug] Plugins: ['SamplePluginIE', 'SamplePluginPP']
[debug] Git HEAD: febff4c11
[debug] Python version 3.8.10 (CPython 64bit) - Linux-5.13.0-28-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4, rtmpdump 2.4
[debug] Optional libraries: secretstorage, sqlite
[debug] Proxy map: {}
[debug] [TwitCastingUser] Extracting URL: https://twitcasting.tv/vazz_kun/show/
[download] Downloading playlist: vazz_kun - Live History
[TwitCastingUser] vazz_kun: Downloading page 1
[TwitCastingUser] playlist vazz_kun - Live History: Downloading 2 videos
[download] Downloading video 1 of 2
[debug] [TwitCasting] Extracting URL: https://twitcasting.tv/vazz_kun/movie/721662456
[TwitCasting] 721662456: Downloading webpage
[TwitCasting] 721662456: Downloading live info
[download] Downloading playlist: Live #721662456
[TwitCasting] playlist Live #721662456: Collected 1 videos; downloading 1 of them
[download] Downloading video 1 of 1
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 721662456-0: Downloading 1 format(s): 0
{"id": "721662456-0", "url": "https://dl193234.twitcasting.tv/tc.vod.v2/v1/streams/721662456.0.2/hls/master.m3u8?k=%2Ftc.vod%2Fv%2F721662456.0.2-1645324857-1645353657-6fcf655d-c993146ca7f7b930", "ext": "mp4", "protocol": "m3u8", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.45 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Encoding": "gzip, deflate", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", "Origin": "https://twitcasting.tv", "Referer": "https://twitcasting.tv/", "Cookie": "hl=en; did=2c84b4ad0a2367182d6582ad8793316c"}, "title": "Live #721662456", "description": "Live #721662456//", "thumbnail": "https://imagegw03.twitcasting.tv/image3/image.twitcasting.tv/mtmbs/image44_1/db2/f8/2b03b1f8.17f1496a26a.jpg", "timestamp": 1645348859, "uploader_id": "vazz_kun", "duration": 1360.377, "view_count": 11, "is_live": false, "n_entries": 1, "_last_playlist_index": 1, "playlist_count": 1, "playlist_index": 1, "playlist_autonumber": 1, "playlist": "Live #721662456", "playlist_id": "721662456", "playlist_title": "Live #721662456", "playlist_uploader": null, "playlist_uploader_id": "vazz_kun", "extractor": "TwitCasting", "webpage_url": "https://twitcasting.tv/vazz_kun/movie/721662456", "webpage_url_basename": "721662456", "webpage_url_domain": "twitcasting.tv", "extractor_key": "TwitCasting", "fulltitle": "Live #721662456", "thumbnails": [{"url": "https://imagegw03.twitcasting.tv/image3/image.twitcasting.tv/mtmbs/image44_1/db2/f8/2b03b1f8.17f1496a26a.jpg", "id": "0"}], "display_id": "721662456-0", "duration_string": "22:40", "upload_date": "20220220", "requested_subtitles": null, "__has_drm": false, "format_id": "0", "format": "0 - unknown", "resolution": null, "dynamic_range": "SDR", "epoch": 1645324858, "_filename": "Live #721662456 [721662456-0].mp4", "filename": "Live #721662456 [721662456-0].mp4", "urls": "https://dl193234.twitcasting.tv/tc.vod.v2/v1/streams/721662456.0.2/hls/master.m3u8?k=%2Ftc.vod%2Fv%2F721662456.0.2-1645324857-1645353657-6fcf655d-c993146ca7f7b930", "_type": "video"}
[download] Finished downloading playlist: Live #721662456
[download] Downloading video 2 of 2
[debug] [TwitCasting] Extracting URL: https://twitcasting.tv/vazz_kun/movie/721661232
[TwitCasting] 721661232: Downloading webpage
[TwitCasting] 721661232: Downloading live info
[download] Downloading playlist: Live #721661232
[TwitCasting] playlist Live #721661232: Collected 1 videos; downloading 1 of them
[download] Downloading video 1 of 1
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 721661232-0: Downloading 1 format(s): 0
{"id": "721661232-0", "url": "https://dl193234.twitcasting.tv/tc.vod.v2/v1/streams/721661232.0.2/hls/master.m3u8?k=%2Ftc.vod%2Fv%2F721661232.0.2-1645324857-1645353657-6fcf655d-9d9bd833c17cae79", "ext": "mp4", "protocol": "m3u8", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.45 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Encoding": "gzip, deflate", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", "Origin": "https://twitcasting.tv", "Referer": "https://twitcasting.tv/", "Cookie": "hl=en; did=2c84b4ad0a2367182d6582ad8793316c"}, "title": "Live #721661232", "description": "Live #721661232//", "thumbnail": "https://imagegw03.twitcasting.tv/image3/image.twitcasting.tv/mtmbs/image242_3/db2/30/2b03ad30.17f14809d7f.jpg", "timestamp": 1645347040, "uploader_id": "vazz_kun", "duration": 1801.117667, "view_count": 10, "is_live": false, "n_entries": 1, "_last_playlist_index": 1, "playlist_count": 1, "playlist_index": 1, "playlist_autonumber": 1, "playlist": "Live #721661232", "playlist_id": "721661232", "playlist_title": "Live #721661232", "playlist_uploader": null, "playlist_uploader_id": "vazz_kun", "extractor": "TwitCasting", "webpage_url": "https://twitcasting.tv/vazz_kun/movie/721661232", "webpage_url_basename": "721661232", "webpage_url_domain": "twitcasting.tv", "extractor_key": "TwitCasting", "fulltitle": "Live #721661232", "thumbnails": [{"url": "https://imagegw03.twitcasting.tv/image3/image.twitcasting.tv/mtmbs/image242_3/db2/30/2b03ad30.17f14809d7f.jpg", "id": "0"}], "display_id": "721661232-0", "duration_string": "30:01", "upload_date": "20220220", "requested_subtitles": null, "__has_drm": false, "format_id": "0", "format": "0 - unknown", "resolution": null, "dynamic_range": "SDR", "epoch": 1645324859, "_filename": "Live #721661232 [721661232-0].mp4", "filename": "Live #721661232 [721661232-0].mp4", "urls": "https://dl193234.twitcasting.tv/tc.vod.v2/v1/streams/721661232.0.2/hls/master.m3u8?k=%2Ftc.vod%2Fv%2F721661232.0.2-1645324857-1645353657-6fcf655d-9d9bd833c17cae79", "_type": "video"}
[download] Finished downloading playlist: Live #721661232
[download] Finished downloading playlist: vazz_kun - Live History
```

You can install latest git master version using pip (`pip install git+https://github.com/yt-dlp/yt-dlp.git`)
